### ISSUE TITLE
Allow to specify alternate server name for postfixadmin

### ISSUE THREAD

**Issue by RomanKisilenko** [zuazo/postfix-dovecot-cookbook#6]
https://github.com/zuazo/postfix-dovecot-cookbook/blob/master/recipes/postfixadmin.rb#L22 explicitly sets the node['postfixadmin']['server_name'].

On some configurations I do want to be able to override it looks like it is not possible at the moment.

**Comment by zuazo** [comment]
If you are having problems with that, you can always overwrite it using `node.override`:

```ruby
node.override['postfixadmin']['server_name'] = 'postfixadmin.example.com'
```

https://docs.chef.io/attributes.html#attribute-types

**Comment by RomanKisilenko** [comment]
Nope, it does not work. What I'm trying to achieve is to set it to nil. node.override['postfixadmin']['server_name'] = nil does not work for me. But when I remove the line mentioned above it starts working just fine.

**Comment by zuazo** [comment]
Why do you want to set it to `nil` instead of other value? What error gives you?

**Comment by RomanKisilenko** [comment]
I'm getting 403 Forbidden for postfixadmin URL due to hostname mismatch. I can work around it, though.
